src/data/causeSpecificJSONData/cats.json

```diff
@@ -0,0 +1,45 @@
+{
+  "path": "cats",
+  "data": {
+    "styles": {
+      "colors": {
+        "primary": "FF6A08",
+        "secondary": "29BEBA",
+        "secondaryShadow": "15AAA6",
+        "background": "FAEFE1"
+      }
+    },
+    "sections": {
+      "landing": {
+        "title": "Open tabs.  Help shelter cats!",
+        "subtitle": "Every tab opened raises money for cats in need of an extra, helping paw. It’s so easy a cat could do it.",
+        "ctaImg": "cat1.png"
+      },
+      "moneyRaised": {
+        "moneyImg": "catsMoneyRaised.png"
+      },
+      "charityIntro": {
+        "title": "Being a shelter cat is hard.",
+        "subTitle": "Our impact is everything to us. And, we’re so proud of the impact we’ve been able to drive, together, thus far.",
+        "introImg1": "introImg1.png",
+        "introImg1Subtext": "More new happy cat owners",
+        "introImg2": "introImg2.png",
+        "introImg2Subtext": "More cats adopted"
+      },
+      "TFACIntro": {
+        "title": "The easiest way to raise money.",
+        "img1": "TFACIntro1.png",
+        "img1Subtext": "Open new tabs on your browser.",
+        "img2": "TFACIntro2.png",
+        "img2Subtext": "Earn Money for cat training programs.",
+        "img3": "TFACIntro3.png",
+        "img3Subtext": "Help get more cats adopted."
+      },
+      "Mission": {
+        "title": "Our Mission",
+        "subtitle": "Help shelter cats get adopted",
+        "text": "We have partnered with The Greater Good Charities' The Jackson Galaxy Project to provide treats to shelter cats for FREE! How can we do this? Through ad revenue! Every time you open a new tab, there will be two small ads in the right corner of your home screen. Companies pay for this space to advertise on our platform, and from there we can put that money to good use by giving it back to an organization doing great work for our furry friends. Tab for Cats is absolutely free for our users, and a great way to make an impact without breaking the bank."
+      }
```

Comment:
I'm still missing 3 sections, but you get the idea

Reply:
Still need these sections for the spec, right?